content/en/docs/concepts/policy/limit-range.md

```diff
@@ -31,17 +31,78 @@ LimitRange object in that namespace.
 
 - The administrator creates one LimitRange in one namespace.
 - Users create resources like Pods, Containers, and PersistentVolumeClaims in the namespace.
-- The `LimitRanger` admission controller enforces defaults and limits for all Pods and Containers that do not set compute resource requirements and tracks usage to ensure it does not exceed resource minimum, maximum and ratio defined in any LimitRange present in the namespace.
-- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) that violates a LimitRange constraint, the request to the API server will fail with an HTTP status code `403 FORBIDDEN` and a message explaining the constraint that have been violated.
-- If a LimitRange is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
+- First, the `LimitRange` admission controller applies default request and limit values for all Pods and Containers that do not set compute resource requirements.
+- Second, the `LimitRange` tracks usage to ensure it does not exceed resource minimum, maximum and ratio defined in any `LimitRange` present in the namespace.
+- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) that violates a `LimitRange` constraint, the request to the API server will fail with an HTTP status code `403 FORBIDDEN` and a message explaining the constraint that have been violated.
+- If a `LimitRange` is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
   requests or limits for those values. Otherwise, the system may reject Pod creation.
-- LimitRange validations occurs only at Pod Admission stage, not on Running Pods.
-
-Examples of policies that could be created using limit range are:
+- `LimitRange` validations occurs only at Pod Admission stage, not on Running Pods.
+
+{{< note >}}
+
+`LimitRange` does not check the consistency of default values it applied. Default value for the limit that is set by `LimitRange` may be less than the request value which will lead to Pod not being scheduleable.
+
+For example, if "LimitRange` is defined as following:
+
+```yaml
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: CpuLimitRange
+  namespace: default
+spec:
+  limits:
+  - default:
+      cpu: 500m
+    defaultRequest:
+      cpu: 500m
+    max:
+      cpu: "1"
+    min:
+      cpu: 100m
+```
+
+The Pod that declares the Request of `700m`, but not the limit, will not be scheduled with the error `Pod "ConflictingCpuSettings" is invalid: spec.containers[0].resources.requests: Invalid value: "700m": must be less than or equal to cpu limit`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ConflictingCpuSettings
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      requests:
+        cpu: 700m
+```
+
+This Pod will be scheduled successfully:
+
+``` yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ConsistentCpuSettings
+spec:
+  containers:
+  - name: nginx
+    image: nginx
```

Comment:
If possible, use an in-project example container image rather than 3rd party software. Also, pin to an image tag if you can.